gen/KubernetesGenerator/templates/Kubernetes.cs.template

```diff
@@ -446,40 +475,26 @@ namespace k8s
                 throw ex;
             }
             // Create Result
-            var _result = new HttpOperationResponse{{GetReturnType operation "<>"}}();
-            _result.Request = _httpRequest;
-            _result.Response = _httpResponse;
-            {{#IfReturnType operation "any"}}
-            // Deserialize Response
-            if ((int)_statusCode == 200 || (int)_statusCode == 201 || (int)_statusCode == 202)
```

Comment:
There is no need to check that _statusCode is 200 - 202. If this was not the case, HttpOperationException would have been thrown on line 475.